File: ios/DriveInCar/Domain/Models/Ranking.swift
import Foundation

struct Ranking: Identifiable, Equatable, Sendable {
    let rankingId: String
    let courseId: String
    let uid: String
    let nickname: String
    let carDisplay: String
    let profileImageId: String
    let timeMs: Int64
    let averageKmh: Double
    let finishedAt: Date
    let flagged: Bool

    var id: String { rankingId }
}

File: android/app/src/main/java/com/driveincar/domain/model/Ranking.kt
package com.driveincar.domain.model

data class Ranking(
    val rankingId: String,
    val courseId: String,
    val uid: String,
    val nickname: String,
    val carDisplay: String,
    val profileImageId: String,
    val timeMs: Long,
    val averageKmh: Double,
    val finishedAtEpochMs: Long,
    val flagged: Boolean,
)
